contracts/AxelarGateway.sol

```diff
@@ -423,6 +434,105 @@ abstract contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
         );
     }
 
+    function transferOperatorship(bytes calldata newOperatorsData, bytes32) external onlySelf {
+        IAxelarAuth(AUTH_MODULE).transferOperatorship(newOperatorsData);
+
+        emit OperatorshipTransferred(newOperatorsData);
+    }
+
+    /********************\
+    |* Internal Methods *|
+    \********************/
+
+    function _unpackLegacyCommands(bytes memory executeData)
+        external
+        pure
+        returns (
+            uint256 chainId,
+            bytes32[] memory commandIds,
+            string[] memory commands,
+            bytes[] memory params
+        )
+    {
+        uint256 signersRole;
+        (chainId, signersRole, commandIds, commands, params) = abi.decode(executeData, (uint256, uint256, bytes32[], string[], bytes[]));
```

Comment:
```suggestion
        (chainId, , commandIds, commands, params) = abi.decode(executeData, (uint256, uint256, bytes32[], string[], bytes[]));
```

Reply:
Applied, my IDE is complaining, but it compiles with no problems